sdk/servicebus/test-resources.bicep

```diff
@@ -35,7 +35,7 @@ resource serviceBusNamespace 'Microsoft.ServiceBus/namespaces@2024-01-01' = {
     zoneRedundant: false
   }
 
-  resource rootSharedAccessKey 'AuthorizationRules@2022-10-01-preview' = {
+  resource rootSharedAccessKey 'AuthorizationRules@2024-01-01' = {
```

Comment:
Just get rid of the versions. They inherit the api-version of the parent resource. It's cleaner and less chance of incompatibility.